src/microbit/pxt_rakettoppskytning/pxt_rakettoppskytning.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "PXT: Rakettoppskytning"
+author: Helene Isnes
+language: nb
+---
+
+
+# Introduksjon {.intro}
+Her skal vi først telle ned fra fem, og så lage en animasjon av en rakett som flyr av gårde.
+
+
+# Steg 1: Nedtelling {.activity}
+
+Først skal vi lage nedtelleren. Du vil at tallet 5 skal vises i et sekund, så 4, så 3 osv. Dette kan du få til ved å bruke en kombinasjon av blokkene `vis tall` og `pause` fra `Basis` kategorien.
+
+
+## Sjekkliste {.check}
+
+- [ ] Få micro:biten til å telle ned fra 5. *Husk at 1000 ms er det samme som 1 sekund*	
+
+- [ ] Koden burde nå se slik ut:
+
+![Bilde som viser nedtelling med `pause` og `vis tall` blokker](nedtelling.png)
+
+# Steg 2: Raketten {.activity}
+
+Nå som vi har fått til nedtelingen trenger vi en rakett. Min rakett ser slik ut:
+
+![Bilde som viser raketten vi skal skyte opp](rakett.png)
+
+Du kan designe raketten din akkurat som du vil.
+
+
+## Sjekkliste {.check}
+
+- [ ] For å skape inntrykket av at raketten skytes opp tegner du ett etter ett bilde der raketten har flyttet seg et steg. Bildene du tegner henger du etter koden i steg 1. 
+
+Eksempel på bilder du kan ende opp med:
+
+![Viser klossen show leds](tom_rakett.png) ![Viser klossen show leds hvor en rad av raketten er synlig](start_rakett.png) ![Viser klossen show leds hvor to rader av raketten er synlig](trinn2_rakett.png) ![Viser klossen show leds hvor tre rader av raketten er synlig](trinn3_rakett.png) ![Viser klossen show leds hvor fire rader av raketten er synlig](trinn4_rakett.png) ![Bilde som viser hele raketten](rakett.png) ![Viser klossen show leds hvor en rad av raketten er forsvunnet](trinn6_rakett.png) ![Viser klossen show leds hvor to rader av raketten er forsvunnet](trinn7_rakett.png) ![Viser klossen show leds hvor tre rader av raketten er forsvunnet](trinn8_rakett.png) ![Viser klossen show leds hvor fire rader av raketten er forsvunnet](trinn9_rakett.png) ![Viser klossen show leds](tom_rakett.png)
```

Comment:
det er plass til tre bilder ved siden av hverandre på nettsiden, så vi burde ha et antall bilder som er delelig på tre for å få det til å se finest ut. vi kan feks fjerne de tomme bildene som er først og sist.